lib/node_modules/@stdlib/math/base/special/besselj0/docs/repl.txt

```diff
@@ -0,0 +1,30 @@
+
+{{alias}}( x )
+    Computes the Bessel function of the first kind of order zero.
+
+    Parameters
+    ----------
+    x: number
+        Input value.
+
+    Returns
+    -------
+    y: number
+        Bessel function of the first kind of order zero.
+
+    Examples
+    --------
+    > var y = {{alias}}( 0.0 )
+    1.0
+    > y = {{alias}}( 1.0 )
+    ~0.765
+    > y = {{alias}}( Number.POSITIVE_INFINITY )
```

Comment:
`y = {{alias}}( {{alias:@stdlib/math/constants/float64-pinf}} )`.

Similarly for negative infinity below.